dask/dataframe/io/tests/test_hdf.py

```diff
@@ -312,10 +312,13 @@ def test_to_hdf_exceptions():
             with pytest.raises(ValueError):
                 a.to_hdf(hdf, '/data_*_*')
 
-
-@pytest.mark.parametrize('get', [dask.async.get_sync,
-                                 dask.threaded.get,
-                                 dask.multiprocessing.get])
+schedulers = [dask.async.get_sync, dask.threaded.get, dask.multiprocessing.get]
+try:
+    import distributed
+    schedulers.append(distributed.Client().get)
+except ImportError:
+    pass
+@pytest.mark.parametrize('get', )
```

Comment:
Thanks, this is correct but irrelevant to the issue I'm having.
See fixed commit. On travis that specific test hangs (10m timeout is reached).